derive/src/utils.rs

```diff
@@ -52,7 +67,7 @@ pub fn const_eval_check_variant_indexes(
 					let mut j = i + 1;
 					while j < len {
 							if indices[i] == indices[j] {
-									::core::panic!("Found Variants that have duplicate indexes. Use different indexes for each variant");
+									::core::panic!(#error_message);
```

Comment:
And then we can generate an even better error message here :P

Reply:
I thought it was impossible, but it is. Done in https://github.com/paritytech/parity-scale-codec/pull/681/commits/876a14ab5d93581f7f9b53e70f624ce4e73dbd24

Also are we ok adding a dependency to `const_format`, if not I can always write the concatenation of strings inlined.
(convertion to array, then calculate the new length in const, then instantiate the new array and copy content + plus special code to format u8).